src/base/utils/string.h

```diff
@@ -63,10 +65,37 @@ namespace Utils::String
 
     QStringList splitCommand(const QString &command);
 
-    QString join(const QList<QStringView> &strings, QStringView separator);
-
     QString fromDouble(double n, int precision);
 
+    template <typename Container>
+    QString joinIntoString(const Container &container, const QString &separator)
```

Comment:
I don't know, maybe it's just me (at least of the two of us) thinking so contextually, but still...
IMO, `String::join` is good enough unlike the case we discussed previously.

Reply:
From my understanding, you insisted that `join` should only operate on string types directly and without a conversion involved. And it should use other name to make it clear that a conversion is involved.

Reply:
> And it should use other name to make it clear that a conversion is involved.

I just consider namespace as part of nested names. OK, forget it.